src/main/java/com/mael/payroll/transactions/AddEmployeeTransaction.java

```diff
@@ -0,0 +1,8 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.PayrollDBFacade;
+
+public interface AddEmployeeTransaction {
+    void execute();
+    void addDB(PayrollDBFacade payrollDB);
```

Comment:
Since my understanding is that all transactions need a database, maybe it's not a bad idea to pass it to the method they all share (`execute()`)? For now I'm gonna pass it to the constructor, maybe that changes in the future.